Gems/AtomLyIntegration/CommonFeatures/Code/Source/CoreLights/DiskLightDelegate.cpp

```diff
@@ -47,53 +47,58 @@ namespace AZ::Render
         return m_shapeBus->GetRadius() * GetTransform().GetUniformScale();
     }
 
-    void DiskLightDelegate::DrawDebugDisplay(const Transform& transform, const Color& /*color*/, AzFramework::DebugDisplayRequests& debugDisplay, bool isSelected) const
+    void DiskLightDelegate::DrawDebugDisplay(const Transform& transform, [[maybe_unused]]const Color&, AzFramework::DebugDisplayRequests& debugDisplay, bool isSelected) const
     {
-        if (isSelected)
+        debugDisplay.PushMatrix(transform);
+        const float radius = GetConfig()->m_attenuationRadius;
+        const float shapeRadius = m_shapeBus->GetRadius();
+
+        auto DrawConicalFrustum = [&debugDisplay](uint32_t numRadiusLines, const Color& color, float brightness, float topRadius, float bottomRadius, float height)
         {
-            debugDisplay.PushMatrix(transform);
-            float radius = GetConfig()->m_attenuationRadius;
+            const Color displayColor = Color(color.GetAsVector3() * brightness);
+            debugDisplay.SetColor(displayColor);
+            debugDisplay.DrawWireDisk(Vector3(0.0, 0.0, height), Vector3::CreateAxisZ(), bottomRadius);
 
-            if (GetConfig()->m_enableShutters)
+            for (uint32_t i = 0; i < numRadiusLines; ++i)
             {
+                float radiusLineAngle = float(i) / numRadiusLines * Constants::TwoPi;
+                debugDisplay.DrawLine(
+                    Vector3(cos(radiusLineAngle) * topRadius, sin(radiusLineAngle) * topRadius, 0),
+                    Vector3(cos(radiusLineAngle) * bottomRadius, sin(radiusLineAngle) * bottomRadius, height)
+                );
```

Comment:
calling cos & sin twice with the same arguments.

Reply:
I'll take it out